lib/tokenizers/tokenizer.ex

```diff
@@ -167,28 +183,40 @@ defmodule Tokenizers.Tokenizer do
     Base.encode32(etag, case: :lower, padding: false)
   end
 
+  @typedoc """
+  Options to set on the loaded tokenizer.
+
+    * `:additional_special_tokens - a list of special tokens to append to the tokenizer.
+      Defaults to `[]`.
+
+    * `:padding` - Override for padding configuration. Currently the only supported
+      value is `:none` to disable padding. By default the configuration is restored
+      from the file.
+
+    * `:truncation` - Override for truncation configuration. Currently the only supported
+      value is `:none` to disable truncation. By default the configuration is restored
+      from the file.
+
+  """
+  @type load_options ::
+          list(
+            {:additional_special_tokens, [String.t() | Tokenizers.AddedToken.t()]}
+            | {:padding, :none}
+            | {:truncation, :none}
+          )
```

Comment:
```suggestion
          [
            additional_special_tokens: [String.t() | Tokenizers.AddedToken.t()],
            padding: :none,
            truncation: :none
          ]
```

Reply:
Just to be sure, typespecs ignore order?

Reply:
Yup, the one I wrote above is rewritten to what you wrote.

Reply:
Oh fantastic!